runtime/devnet/build.rs

```diff
@@ -1,12 +1,15 @@
-#[cfg(feature = "std")]
+#[cfg(all(feature = "std", feature = "metadata-hash"))]
```

Comment:
I see this line in the base parachain:
```#[docify::export(template_enable_metadata_hash)]```

Not sure what it is for and whether we want it. Not something that should stop this upgrade though.

Reply:
I think that line of code is just to export the documentation so don't know if we want that. Hence, I did not add any thing related to `docify`

Reply:
Sounds good!